core/src/main/java/org/jdbi/v3/Query.java

```diff
@@ -146,8 +146,8 @@
      * @see Handle#registerRowMapper(RowMapperFactory)
      * @see Handle#registerRowMapper(RowMapper)
      */
-    private <T> Query<T> mapTo(Type resultType) {
-        return this.map(new RegisteredRowMapper<>(resultType, mappingRegistry));
+    public Query mapTo(Type resultType) {
```

Comment:
The `<T>` was ok while the method was private, but since I had to make it part of the API to bring sqlobject "up to code" on enhanced generics, I felt it was better to underpromise in this case.